config/200-clusterrole.yaml

```
@@ -10,7 +10,7 @@ metadata:
 aggregationRule:
   clusterRoleSelectors:
   - matchLabels:
-      servicebinding.io/controller: "true"
+      bindings.labs.vmware.com/admin: "true"
```

we still need to match the `servicebinding.io/controller: "true"` label otherwise we won't be able to bind new service and workload types.

```suggestion
      bindings.labs.vmware.com/admin: "true"
  - matchLabels:
      servicebinding.io/controller: "true"
```